com.unity.ml-agents/Runtime/Communicator/GrpcExtensions.cs

```diff
@@ -80,6 +99,25 @@ public static BrainParametersProto ToProto(this BrainParameters bp, string name,
             return brainParametersProto;
         }
 
+        /// <summary>
+        /// Convert a BrainParametersProto to a BrainParameters struct.
+        /// </summary>
+        /// <param name="bpp">An instance of a brain parameters protobuf object.</param>
+        /// <returns>A BrainParameters struct.</returns>
+        public static BrainParameters ToBrainParameters(this BrainParametersProto bpp)
```

Comment:
Just moved this closer to the BrainParameters -> BrainParametersProto conversion